src/styles/SearchForm.css

```diff
@@ -0,0 +1,20 @@
+.search-form {
+  margin: auto;
+  margin-bottom: 2.5%;
```

Comment:
You have to be careful with when to use percentages when it comes to CSS, they are not always appropriate. Why 2.5%? Why not just a static value?